python_modules/dagster/dagster_tests/storage_tests/utils/run_storage.py

```diff
@@ -1730,6 +1730,38 @@ def test_backfill_simple_job_name_filtering(self, storage: RunStorage):
         )
         assert len(backfills_for_job) == 0
 
+    def test_backfill_id_filtering(self, storage: RunStorage):
+        origin = self.fake_partition_set_origin("fake_partition_set")
+        backfills = storage.get_backfills()
+        assert len(backfills) == 0
+
+        backfill = PartitionBackfill(
+            "backfill_1",
+            partition_set_origin=origin,
+            status=BulkActionStatus.REQUESTED,
+            partition_names=["a", "b", "c"],
+            from_failure=False,
+            tags={},
+            backfill_timestamp=time.time(),
+        )
+        storage.add_backfill(backfill)
```

Comment:
nitpick, but could we also add another backfill with a different id just to make sure this filter is actually applied?